pages/facts.html

```diff
@@ -4,9 +4,97 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../style/style.css">
+  <link rel="stylesheet" href="../style/facts.css">
+  <title>Facts about plants </title>
+  <link rel="icon" href="../assets/plant-icon.png" type="image/png">
 </head>
 <body>
-  
+  <header>
+    <h1>Plant Party Fansite</h1>
+    <nav>
+      <ul>
+        <li>
+          <a class="menu-link" href="../index.html" target="_blank">Home</a>
+        </li>
+        <li>
+          <a class="menu-link" href="care.html" target="_blank">Care</a>
+        </li>
+        <li>
+          <a class="menu-link" href="#">Facts</a>
+        </li>
+        <li>
+          <a class="menu-link" href="gallery.html" target="_blank">Gallery</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h2>🌱 Funny facts about plants and not only 🌱</h2>
+      <img src="../assets/Plants.jpg" width="400" alt="Plants"/>
+      <p>
```

Comment:
I question whether the Paragraph element is the most appropriate element to use here because: "Paragraphs are usually represented in visual media as blocks of text separated from adjacent blocks by blank lines and/or first-line indentation".

See docs [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p).

Perhaps, it can be removed completely or replaced with a more semantic tag.